src/react-image-lightbox.js

```diff
@@ -1313,7 +1339,25 @@ class ReactImageLightbox extends Component {
         imageStyle.cursor = 'move';
       }
 
-      if (bestImageInfo === null) {
+      // when error on one of the loads then push custom error stuff
+      if (
+        bestImageInfo === null &&
+        Object.values(loadErrorStatus).includes(true)
```

Comment:
Pretty sure `Object.values` and `includes` might be iffy when it comes to browser compatibility. At worst I want to support IE 11, with barebones / fatal-error-free functionality in IE 9.

Reply:
I will find a compatible solution.